### PR TITLE
Update GPU availability in the EGI Federated Cloud

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

This PR updates the information published in: https://docs.egi.eu/users/compute/cloud-compute/gpgpu/

---

**Related issue :**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
A preview is available at: https://docs.egi.eu/documentation/448/users/compute/cloud-compute/gpgpu/

File: content/en/users/compute/cloud-compute/gpgpu/_index.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ The table below summarises the available options:
...
-| IISAS-FedCloud | up to 2 NVIDIA Tesla K20m              | g1.c08r31-K20m, g1.c16r62-2xK20m                                                                                                                                                           | vo.access.egi.eu, eosc-synergy.eu, enmr.eu, training.egi.eu | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
+| IISAS-FedCloud | up to 8 NVIDIA Tesla K20m              | g1.c08r30-K20m, g1.c16r60-2xK20m                                                                                                                                                           | vo.access.egi.eu, eosc-synergy.eu, enmr.eu, training.egi.eu | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
 | IFCA-LCG2      | up to 2 NVIDIA T4, up to 2 NVIDIA V100 |                                                                                                                                                                                            |                                                            | Pay-per-use                                                                           |
```
Comment (alvarolopez) [inline content/en/users/compute/cloud-compute/gpgpu/_index.md:24]: This is not updated, please check: https://confluence.ifca.es/display/IC/Cloud+Compute+Flavors

We can provide up to 20 V100 and up to 80 T4

**Comment by sebastian-luna-valero** [comment]
Also, we need to add: https://docs.cloud.muni.cz/cloud/gpus/

File: content/en/users/compute/cloud-compute/gpgpu/_index.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ The table below summarises the available options:
...
-| CESNET-MCC     | up to 2 NVIDIA T4                      | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com       | Sponsored, conditions to be negotiated                                                |
+| CESNET-MCC     | up to 2 NVIDIA Tesla T4, up to 4 NVIDIA A40, up to 2 GeForce RTX 1080/2080 experimental use only | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.32core-256ram-nvidia-t4-single-gpu, hpc.64core-238ram-nvidia-t4, hpc.64core-512ram-nvidia-t4, meta-hdm.16core-120ram-nvidia-a40, meta-hdm.64core-485ram-nvidia-a40-quad | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com, cryoem.instruct-eric.eu, fusion, icecube, vo.carouseldancing.org, vo.pangeo.eu, vo.neanias.eu, umsa.cerit-sc.cz, vip, vo.notebooks.egi.eu, vo.emphasisproject.eu, vo.inactive-sarscov2.eu | Sponsored, conditions to be negotiated |
```
Comment (Pansanel) [inline content/en/users/compute/cloud-compute/gpgpu/_index.md:26]: IN2P3-IRES is also providing GPU. Can the following line be added:
| IN2P3-IRES     | up to 1 NVIDIA Tesla T4 per VM, up to 1 NVIDIA A40 per VM, up to 4 GeForce RTX 2080 per VM (experimental use only) | g1.xlarge-4xmem, g2.xlarge-4xmem, g4.xlarge-4xmem  | biomed, vo.emphasisproject.eu, vo.france-grilles.fr | Sponsored, conditions to be negotiated |

**Review by enolfc — APPROVED** [review]
👍 Thanks @sebastian-luna-valero and sites!

**Comment by gwarf** [comment]
@sebastian-luna-valero please review and resolve the conversation so that it's marked as ready for being merged, thanks!

File: content/en/users/compute/cloud-compute/gpgpu/_index.md (hunk 1 above)
Comment (sebastian-luna-valero) [reply content/en/users/compute/cloud-compute/gpgpu/_index.md]: This has been updated, thanks!

File: content/en/users/compute/cloud-compute/gpgpu/_index.md (hunk 2 above)
Comment (sebastian-luna-valero) [reply content/en/users/compute/cloud-compute/gpgpu/_index.md:27]: This has been updated, thanks!

**Pull request merged**